extensions/amp-audio/validator-amp-audio.protoascii

```diff
@@ -28,10 +28,14 @@ attr_lists: { # amp-audio attributes that are common to both AMP and A4A format.
   name: "amp-audio-common"
   attrs: { name: "album" }
   attrs: { name: "artist" }
+  attrs: { name: "artwork" }
   attrs: { name: "controls" }
   attrs: { name: "loop"  value: "" }
   attrs: { name: "muted" value: "" }
-  attrs: { name: "poster" }
+  attrs: {
+    # deprecated use artwork instead
```

Comment:
I believe there is an official validator way of deprecating. @honeybadgerdontcare